docs/telemetry/COLLECTED_DATA.md

```diff
@@ -0,0 +1,40 @@
+# Collected telemetry data
```

Comment:
@whizzzkid @lidel first demo of what COLLECTED_DATA.md should look like as defined from ipfs-shipyard/ignite-metrics#35.  Lmk what you think